### ISSUE TITLE
docs build broken for 1.3.0a1

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#6598]
```
Running Sphinx v1.8.6

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/checkouts/1.3.0a1/docs/conf.py", line 18, in <module>
    from borg import __version__ as sw_version
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/checkouts/1.3.0a1/src/borg/__init__.py", line 3, in <module>
    from ._version import version as __version__
ModuleNotFoundError: No module named 'borg._version'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/cmd/build.py", line 303, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/application.py", line 201, in __init__
    self.config = Config.read(self.confdir, confoverrides or {}, self.tags)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/config.py", line 203, in read
    namespace = eval_config_file(filename, tags)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/config.py", line 380, in eval_config_file
    raise ConfigError(msg % traceback.format_exc())
sphinx.errors.ConfigError: There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/checkouts/1.3.0a1/docs/conf.py", line 18, in <module>
    from borg import __version__ as sw_version
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/checkouts/1.3.0a1/src/borg/__init__.py", line 3, in <module>
    from ._version import version as __version__
ModuleNotFoundError: No module named 'borg._version'


Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/envs/1.3.0a1/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/checkouts/1.3.0a1/docs/conf.py", line 18, in <module>
    from borg import __version__ as sw_version
  File "/home/docs/checkouts/readthedocs.org/user_builds/borgbackup/checkouts/1.3.0a1/src/borg/__init__.py", line 3, in <module>
    from ._version import version as __version__
ModuleNotFoundError: No module named 'borg._version'
```

**Comment by ThomasWaldmann** [comment]
This does work, into a fresh virtual env:
```
pip install borgbackup==1.3.0a1
```

**Comment by ThomasWaldmann** [comment]
rtfd master build is broken since "a week" (f*cking "human readable" time stamps), that should be about 2022-04-08.

**Comment by ThomasWaldmann** [comment]
https://github.com/borgbackup/borg/commit/9642ace95324cb7800004afcd63a345116dfbfbb this change maybe?

**Comment by ThomasWaldmann** [comment]
Check if the same problem exists for 1.2-maint.

**Comment by ThomasWaldmann** [comment]
No fix needed for 1.2-maint: i was cautious there and setuptools_scm is still used the old way there == in setup.py, not pyproject.toml.

**Comment by ThomasWaldmann** [comment]
I'll also add setuptools_scm to pyproject.toml in 1.2-maint though, so it'll get pulled via that mechanism also (so it is **present**). Configuration stays in setup.py for now.
